angular-portfolio/src/app/award-detail/award-detail.component.html

```diff
@@ -0,0 +1,16 @@
+<div *ngIf="award">
+    <div class="flex-container">
+        <div *ngFor="let awd of awards" classs="photo"
```

Comment:
I'd recommend changing this to "let award of awards", the abbreviation is less clear.

Reply:
I already have an "award" in the namespace. I'll switch it to awardloop to prevent ambiguity.